packages/headless/src/app/commerce-engine/commerce-engine.ts

```diff
@@ -34,6 +35,7 @@ const commerceEngineReducers = {
   commercePagination: paginationReducer,
   commerceSort: sortReducer,
   facetOrder: facetOrderReducer,
+  facetSearchSet: specificFacetSearchSetReducer,
```

Comment:
The state will be the same regardless of whether it's a category or regular facet, so we can use this reducer.